### PR TITLE
SCMOD-7628: Logging updates

### PR DESCRIPTION
**Pull request by kusumaghoshdastidar**
Jira: https://portal.digitalsafe.net/browse/SCMOD-7628

Dev builds: http://sou-jenkins2.hpeswlab.net/job/CAFAudit/view/Developer/job/CAFAudit~audit-service~SCMOD-7628~CI/

### CODE REVIEW

File: caf-audit-binding-elasticsearch/src/test/java/com/hpe/caf/auditing/elastic/ElasticAuditIT.java [hunk 1]
```diff
@@ -83,6 +83,8 @@ public static void setup() throws Exception
     {
         // Test the Auditing library in elasticsearch mode
         System.setProperty("CAF_AUDIT_MODE", "elasticsearch");
+        System.setProperty("docker.host.address", "larry-es01.swinfra.net");
+        System.setProperty("es.port", "9200");
```
Comment (dermot-hardy) [inline caf-audit-binding-elasticsearch/src/test/java/com/hpe/caf/auditing/elastic/ElasticAuditIT.java]: Remove these lines.

File: caf-audit-service-container/pom.xml [hunk 2]
```diff
@@ -258,7 +261,7 @@
                                     <CAF_ELASTIC_PORT_VALUE>9200</CAF_ELASTIC_PORT_VALUE>
                                     <CAF_ELASTIC_NUMBER_OF_SHARDS>5</CAF_ELASTIC_NUMBER_OF_SHARDS>
                                     <CAF_ELASTIC_NUMBER_OF_REPLICAS>1</CAF_ELASTIC_NUMBER_OF_REPLICAS>
-                                    <CAF_LOG_LEVEL>DEBUG</CAF_LOG_LEVEL>
+                                    <CAF_LOG_LEVEL>TRACE</CAF_LOG_LEVEL>
```
Comment (dermot-hardy) [inline caf-audit-service-container/pom.xml]: Does this really need to change?

**Review by dermot-hardy — CHANGES_REQUESTED** [review]

File: caf-audit-binding-elasticsearch/src/test/java/com/hpe/caf/auditing/elastic/ElasticAuditIT.java (hunk 1 above)
Comment (kusumaghoshdastidar) [reply caf-audit-binding-elasticsearch/src/test/java/com/hpe/caf/auditing/elastic/ElasticAuditIT.java]: Removed

File: caf-audit-service-container/pom.xml (hunk 2 above)
Comment (kusumaghoshdastidar) [reply caf-audit-service-container/pom.xml]: No. Reverted

**Review by dermot-hardy — APPROVED** [review]
Did you get the log4j2 issue sorted out that you were showing me yesterday?

**Comment by kusumaghoshdastidar** [comment]
> Did you get the log4j2 issue sorted out that you were showing me yesterday?

No.

**Comment by dermot-hardy** [comment]
It should really be fixed as part of this PR.

**Comment by kusumaghoshdastidar** [comment]
> It should really be fixed as part of this PR.

Ok

**Pull request merged**